docs/crd_subnamespace.md

```diff
@@ -28,3 +30,14 @@ SubNamespaceList contains a list of SubNamespace
 | items |  | [][SubNamespace](#subnamespace) | true |
 
 [Back to Custom Resources](#custom-resources)
+
+#### SubNamespaceSpec
+
+SubNamespaceSpec defines the desired state of SubNamespace
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| labels | Labels are the labels to be propagated to the sub-namespace | map[string]string | false |
+| annotations | Annotations are the annotations to be propagated to the sub-namespace. | map[string]string | false |
```

Comment:
I don't think `propagate` is an appropriate description.
Because it just sets the labels/annotations of SubNamespace to the target Namespace.
Those labels/annotations are not propagated to the descendant namespaces.

Reply:
@zoetrope point is valid.
But it would be better to change the implementation to actually propagate
the labels/annotations from SubNamespace to the target Namespace so that
a tenant user can edit Namespace labels/annotations by editing SubNamespaces.

What do you think? @zoetrope @bells17

Reply:
@ymmt2005 
Perhaps my point was misleading.
The current implementation propagates the labels/annotations from SubNamespace to the target Namespaces.

Reply:
@zoetrope 
> The current implementation propagates the labels/annotations from SubNamespace to the target Namespaces.

I don't think so.

Take a look at this:
https://github.com/cybozu-go/accurate/blob/add-labels-and-annotations-field2/controllers/subnamespace_controller.go#L95

The SubNamespace controller only creates the Namespace and never updates it afterwards.

Reply:
@ymmt2005 
Sorry, I understood.

@bells17 
Please fix it to update the labels and annotations in reconcile.